python/cudf/cudf/_lib/nvtx/_lib/lib.pyx

```diff
@@ -0,0 +1,64 @@
+from nvtx._lib.lib cimport *
+
+
+def initialize():
+    nvtxInitialize(NULL)
+
+
+cdef class EventAttributes:
```

Comment:
We should define this class in a header instead so someone can `cimport` it if desired